test/compile_test/instantiate.hpp

```diff
@@ -409,9 +467,21 @@ void instantiate(RealType)
 
    boost::math::unchecked_bernoulli_b2n<RealType>(i);
    boost::math::bernoulli_b2n<RealType>(i);
-   boost::math::bernoulli_b2n<RealType>(i, i, &v1);
+   {
+      RealType* v1_array = new RealType[i];
+
+      boost::math::bernoulli_b2n<RealType>(i, i, &v1_array[0U]);
+
+      delete [] v1_array;
+   }
    boost::math::tangent_t2n<RealType>(i);
-   boost::math::tangent_t2n<RealType>(i, i, &v1);
+   {
+      RealType* v1_array = new RealType[i];
```

Comment:
Ditto.

Reply:
What do you mean? Like std::vector?

Reply:
I was going to suggest a Boost solution, but these days it's spelled std::unique_ptr: https://en.cppreference.com/w/cpp/memory/unique_ptr